lib/fastlane/plugin/wpmreleasetoolkit/actions/configure/configure_apply_action.rb

```diff
@@ -54,39 +54,43 @@ def self.prepare_repository
       ### Check with the user whether we should overwrite the file, if it exists
       ###
       def self.apply_file(file_reference, force)
```

Comment:
Unfortunately, the diff is noisy. The only change here was wrapping the logic in a `begin – rescue` block, to gracefully handle the expected error raised if a file is not ignored without dumping the stacktrace on the user

Reply:
"Ignore whitespace" to the rescue! 
![image](https://user-images.githubusercontent.com/216089/116390529-6c09d100-a81e-11eb-88d6-a97b8cb1745e.png)